_layouts/formula-linux.html

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+permalink: :title
+---
+{%- assign full_name = page.title -%}
+{%- assign name = full_name | remove: "@" | remove: "." | replace: "+", "_" -%}
+{%- assign f = site.data.formula-linux[name] -%}
+<h2>{{ f.name }}</h2>
+{%- if f.aliases.size > 0 -%}
+<p>
+    Also known as:
+    <strong>{{ f.aliases | join: "</strong>, <strong>" }}</strong>
+</p>
+{%- endif -%}
+{%- if f.oldname -%}
+<p>Formerly known as: <strong>{{ f.oldname }}</strong></p>
+{%- endif -%}
+<p>{{ f.desc | xml_escape }}</p>
+<p>
+    <a href="{{ f.homepage }}">{{ f.homepage }}</a>
+</p>
+
+<p><a href="{{ site.baseurl }}/api/formula-linux/{{ f.name }}.json"><code>/api/formula-linux/{{ f.name }}.json</code> (JSON API)</a></p>
+<p><a target="_blank" href="https://github.com/Homebrew/linuxbrew-core/blob/master/Formula/{{ f.name }}.rb">Linux formula code</a> on GitHub</p>
+
+<p>Current versions:</p>
+<table>
+    <tr>
+        <td>stable</td>
+        <td>✅</td>
+        <td>{{ f.versions.stable }}</td>
+    </tr>
+    {%- if f.versions.devel -%}
+    <tr>
+        <td>devel</td>
+        <td>🛠</td>
+        <td>{{ f.versions.stable }}</td>
+    </tr>
+    {%- endif -%}
+    {%- if f.versions.head -%}
+    <tr>
+        <td>head</td>
+        <td>⚡️</td>
+        <td>{{ f.versions.head }}</td>
+    </tr>
+    {%- endif -%}
+    {%- if f.versions.bottle -%}
+    <tr>
+        <td>bottle</td>
+        <td>🍾</td>
+        <td>
+            {%- for b in f.bottle.stable.files -%}
+            {{ b[0] }}
+            {%- unless forloop.last -%}, {% endunless %}
+            {%- endfor -%}
+        </td>
+    </tr>
+    {%- endif -%}
+</table>
+
+{%- include formulae.html formulae=f.versioned_formulae description="Other versions" -%}
```

Comment:
```suggestion
{%- include formulae-linux.html formulae=f.versioned_formulae description="Other versions" -%}
```
As above, either create `_includes/formulae-linux.html` or update `_includes/formulae.html` to use a page variable passed from here.